lib/taiko.js

```diff
@@ -1018,6 +1018,7 @@ module.exports.focus = async (selector, options = {}) => {
  * @param {number} [options.waitForStart=100] - wait for navigation to start. Accepts time in milliseconds.
  * @param {number} [options.timeout=30000] - DEPRECATED timeout option, use navigationTimeout instead.
  * @param {number} [options.navigationTimeout=30000] - Navigation timeout value in milliseconds for navigation after click.
+ * @param {boolean} [options.hidden=false] - Prevent given text from being written to log output.
```

Comment:
The options.hidden has a different meaning in some APIs. It means that the element to perform the action is hidden.
To avoid confusion, its better we use a different name here. In my opinion, `options.hideText` is a good name to use.

Reply:
@nehashri thanks for the quick look! I struggled to come up with a name for it and was definitely iffy on `hidden`. I'll change it to your suggestion!